### DISCUSSION TITLE
How to get generated coordinates (bounding box) in Pixel Shader

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#294] [Q&A]
I did not see any way with the default pipeline to get the generated coordinates of an object because here does not seem to be a way to access the bounding box of an object.

**Comment by pragma37** [comment]
This is not supported at the moment. Are generated coordinates actually useful?

**Reply by Kolupsy** [reply]
I would say so as it is the only option to get gradients that are always mapped 0-1 across the entire object. I guess it is a judgement call if it makes sense to implement but its not too big of a deal if its not in malt

**Reply by pragma37** [reply]
I think that's a reasonable use case. I'll look into it.

Just keep in mind this will not take displacement into account.
